src/main/scala/chisel3/ModuleChoice.scala

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3
+
+import scala.language.experimental.macros
+import scala.collection.immutable.ListMap
+
+import chisel3.FixedIOBaseModule
+import chisel3.experimental.{BaseModule, SourceInfo}
+import chisel3.internal._
+import chisel3.internal.Builder.pushCommand
+import chisel3.internal.firrtl._
+import chisel3.internal.sourceinfo.InstChoiceTransform
+import chisel3.internal.{Builder}
```

Comment:
Nit: please avoid wildcard imports. I don't think there are that many things imported here?

Also, the following is sufficient, no curly braces:

```
import chisel3.internal.Builder
```